server/test/unit/vitalsTest/checkForInternalProblemsTest.js

```diff
@@ -0,0 +1,215 @@
+/* eslint-disable no-plusplus */
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const rewire = require('rewire')
+
+// In-house dependencies
+const { ALERT_TYPE, factories, helpers } = require('brave-alert-lib')
+const db = require('../../../db/db')
+const { locationDBFactory, sessionDBFactory } = require('../../../testingHelpers')
+
+const vitals = rewire('../../../vitals')
+
+// Configure Chai
+use(sinonChai)
+
+const sandbox = sinon.createSandbox()
+
+const maxStillnessAlerts = helpers.getEnvVar('MAX_STILLNESS_ALERTS')
+
+describe('vitals.js integration tests: checkForInternalProblems', () => {
+  /* eslint-disable no-underscore-dangle */
+  beforeEach(async () => {
+    await db.clearTables()
+    sandbox.stub(helpers, 'log')
+    sandbox.stub(helpers, 'logSentry')
+  })
+
+  afterEach(async () => {
+    await db.clearTables()
+    sandbox.restore()
+  })
+
+  describe('no locations in the database', () => {
+    beforeEach(async () => {
+      await vitals.checkForInternalProblems()
+    })
+
+    it('should not send any alerts to Sentry', () => {
+      expect(helpers.logSentry).to.not.have.been.called
+    })
+
+    it('should not log any errors', () => {
+      expect(helpers.log).to.not.be.called
+    })
+  })
+
+  describe('for a location that has no previous sessions', () => {
+    beforeEach(async () => {
+      // Insert a single location with no sessions
+      const client = await factories.clientDBFactory(db, { isActive: true })
+      this.testLocation = await locationDBFactory(db, {
+        clientId: client.id,
+        isActive: true,
+      })
+      await vitals.checkForInternalProblems()
+    })
+
+    it('should not send any alerts to Sentry', () => {
+      expect(helpers.logSentry).to.not.have.been.called
+    })
+
+    it('should not log any errors', () => {
+      expect(helpers.log).to.not.be.called
+    })
+  })
+
+  describe('should not log to Sentry if exactly maxStillnessAlerts have occurred in 24 hours at a location', () => {
+    beforeEach(async () => {
+      // Insert a single location and maxStillnessAlerts sessions + 1
```

Comment:
This commend doesn't seem to match what the test is doing. You actually have exactly `maxStillnessAlerts` sessions, right?

Reply:
Oh yes! Thank you for pointing that comment out, I need to go fix it in my other tests haha